server/src/test/java/org/elasticsearch/index/mapper/RootObjectMapperTests.java

```diff
@@ -28,6 +28,11 @@
 import java.util.Arrays;
 
 public class RootObjectMapperTests extends ESSingleNodeTestCase {
+    @Override
+    protected boolean enableWarningsCheck() {
+        // disable warning checks as deprecated patterns are used to compare Joda vs Java results (y - year and Z zone offset)
```

Comment:
I get doing this for date-specific tests, but there's a *lot* of classes that have the warnings check disabled here, including some very broad ones. We should be careful about doing this so widely, although I could see an argument that this is okay if it would be a lot of work to use `assertWarnings` everywhere given these will only be going into 6.8, which should pretty much only be getting bugfixes at this point.

Reply:
where possible I used explicitly assertWarnings about these patterns, but often it was just not feasible and would require adding lot of them.
and agree, that is only going to be in 6.8 so hopefully would not reduce confidence in these tests